internal/docgen/docgen.go

```diff
@@ -42,26 +36,6 @@ func (g documentationGenerator) Generate(destinationDir string) error {
 	return nil
 }
 
-// trivyVersion returns the current version of the trivy library used.
-func trivyVersion() (*string, error) {
```

Comment:
This method is nice but is not easily testable. Hence the move to the environment variable.